src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilter.java

```diff
@@ -59,6 +59,7 @@
             Arrays.asList("ReturnCountCheck",
                     "MethodCountCheck",
                     "AnonInnerLengthCheck",
+                    "DescendantTokenCheck",
```

Comment:
please send PR to make this list empty, and place comment about it 
"place in this collection all standard checks that support context #xxxxx"

You can add name to issue, one by one, or we can search in xml files what checks should be there.

Rationale: hardcoded name always worry me, and force me to recheck.

Reply:
done